css/Helpers/_helpers.scss

```diff
@@ -0,0 +1,10 @@
+@mixin square($width) {
+    width: $width;
+    height: $width;    
```

Comment:
height nie powinno przyjmowac zmiennej "width", caly mixin powinien przyjmowac 2 zmienne: width i height

natomiast mozesz do zmiennej height przypisac wartosc domyslna :width, wtedy jak bedziesz chcial osiagnac idealny kwadrat to bedzie mozna podac 1 wartosc, omawialismy to na zajeciach wiec jest na nagraniu